mesheryctl/pkg/utils/testing.go

```diff
@@ -127,6 +139,21 @@ func SetupContextEnv(t *testing.T) {
 func StartMockery(t *testing.T) {
 	// activate http mocking
 	httpmock.Activate()
+
+	// get current directory
+	_, filename, _, ok := runtime.Caller(0)
+	if !ok {
+		t.Fatal("problems recovering caller information")
```

Comment:
This error message is bit confusing. Probably, we could mention - "Not able to get current working directory", as our purpose of calling runtime.Caller(0) is to get CWD.